### PR TITLE
[RP-31] Make Class relate with User

### PR DESCRIPTION
**Pull request by Nine0512**
https://nine0512.atlassian.net/jira/software/projects/RP/boards/1?selectedIssue=RP-31

### CODE REVIEW

File: src/main/kotlin/com/rooplor/classcraftbackend/controllers/ClassController.kt [hunk 1]
```diff
@@ -98,4 +98,10 @@ class ClassController
         fun removeClass(
             @PathVariable id: String,
         ): Unit = service.deleteClass(id)
+
+        @Operation(summary = "Find classes by owners")
+        @GetMapping("/owners")
+        fun findClassByOwners(
+            @RequestParam owners: List<String>,
+        ): List<ClassListDTO> = listMapper.mapList(service.findClassByOwners(owners), ClassListDTO::class.java, modelMapper)
```
Comment (c3bosskung) [inline src/main/kotlin/com/rooplor/classcraftbackend/controllers/ClassController.kt:106]: what about create class? It should modify to add user when create?

Comment (Nine0512) [reply src/main/kotlin/com/rooplor/classcraftbackend/controllers/ClassController.kt:106]: I updated classDTO already for pass user who created this class

File: src/main/kotlin/com/rooplor/classcraftbackend/services/AuthService.kt [hunk 2]
```diff
@@ -45,6 +45,12 @@ class AuthService(
         return authentication?.name
     }
 
+    fun getAuthenticatedUserDetails(): User? {
+        val username = getAuthenticatedUser()
+        print(username)
```
Comment (c3bosskung) [inline src/main/kotlin/com/rooplor/classcraftbackend/services/AuthService.kt:50]: Can we remove this logging to avoid confuse log in future?

**Review by c3bosskung — COMMENTED** [review]
add some comment

File: src/main/kotlin/com/rooplor/classcraftbackend/services/AuthService.kt (hunk 2 above)
Comment (Nine0512) [reply src/main/kotlin/com/rooplor/classcraftbackend/services/AuthService.kt]: Sorry I forgot to remove debug log.now removed already.

**Review by c3bosskung — APPROVED** [review]
LGMT

**Pull request merged**